frontend/README.md

```diff
@@ -0,0 +1,7 @@
+# PRISM React App
+
+## Run a single test 
+
+```
+npm run test -- -t 'Test name'
```

Comment:
I am not sure this is needed. But if you think it's useful, we use yarn in this repo so let's use that as the pm. :)

Reply:
I will remove it